modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/upgrade/v3_2_0/UpgradeDLFileEntryType.java

```diff
@@ -14,9 +14,221 @@
 
 package com.liferay.document.library.internal.upgrade.v3_2_0;
 
+import com.liferay.document.library.kernel.model.DLFileEntryMetadata;
+import com.liferay.document.library.kernel.model.DLFileEntryType;
+import com.liferay.dynamic.data.mapping.model.DDMStructure;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.portal.kernel.dao.jdbc.AutoBatchPreparedStatementUtil;
+import com.liferay.portal.kernel.security.permission.ResourceActionsUtil;
+import com.liferay.portal.kernel.service.ResourceLocalServiceUtil;
+import com.liferay.portal.kernel.util.PortalUtil;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.portal.kernel.uuid.PortalUUIDUtil;
+
+import java.sql.Date;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
 /**
  * @author Adolfo Pérez
  */
 public class UpgradeDLFileEntryType
 	extends com.liferay.portal.upgrade.v7_3_x.UpgradeDLFileEntryType {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		super.doUpgrade();
+
+		try (PreparedStatement ps1 = connection.prepareStatement(
+				StringBundler.concat(
+					"select companyId, fileEntryTypeId, fileEntryTypeKey, ",
```

Comment:
In this query, all SQL keywords are in lowercase, but in others in uppercase. Take a look at other upgrade classes an choose the more common one.